OrbitCore/ConnectionManager.cpp

```diff
@@ -357,15 +357,15 @@ void ConnectionManager::SetupClientCallbacks() {
       });
 }
 
-void ConnectionManager::SendProcesses(TcpEntity* tcp_entity) {
+void ConnectionManager::SendProcesses(std::shared_ptr<TcpEntity> tcp_entity) {
```

Comment:
I do not think this is necessary, we do now want to impose unnecessary restrictions on the way TcpEntry is scoped. I would avoid using shared_ptrs unless this is completely unavoidable.

Current situation where shared_ptr is used because there is no well established ownership of an object is not pattern we should reuse.

Reply:
Maybe keep a pointer here... the method does not store tcp_entity anywhere.

Reply:
done